specs/protocol/README.md

```diff
@@ -319,19 +276,29 @@ Currency amounts have type `DecimalString`, which is string containing a decimal
+| `payinMethod`  | [`SelectedPaymentMethod`](#selectedpaymentmethod) | Y        | Specify which payment method to send payin currency.                                                                        |
+| `payoutMethod` | [`SelectedPaymentMethod`](#selectedpaymentmethod) | Y        | Specify which payment method to receive payout currency.                                                                    |
+
+#### Unsigned Fields
+In many instances, an RFQ may include Personally Identifiable Information (PII) or Payment Card Industry (PCI) data, either embedded within the `claims` being presented or within the `paymentDetails` of `payinMethod` or `payoutMethod` (such as card details, phone numbers, full names, etc.).
+
+To safeguard privacy, the signature deliberately excludes these fields during the signing process. Specifically, the fields `claims`, `payinMethod.paymentDetails`, and `payoutMethod.paymentDetails` are removed from the RFQ before signing. To ensure the integrity of the signature, each excluded field undergoes a hashing process. The resulting hash is then placed in the corresponding field, namely `claimsHash`, `payinMethod.paymentDetailsHash`, and `payoutMethod.paymentDetailsHash`. This allows PFIs to selectively omit the unsigned fields when storing or transmitting the message.
+
+For further information on supported hashing algorithms, please refer to [digest](#digest) section.
 
 #### `SelectedPaymentMethod`
-| field            | data type | required | description                                                                                       |
-| ---------------- | --------- | -------- | ------------------------------------------------------------------------------------------------- |
-| `kind`           | string    | Y        | Type of payment method (i.e. `DEBIT_CARD`, `BITCOIN_ADDRESS`, `SQUARE_PAY`)                       |
-| `paymentDetails` | object    | N        | An object containing the properties defined in an Offering's `requiredPaymentDetails` json schema |
+| field                | data type | required | description                                                                                                                             |
+|----------------------|-----------|----------|-----------------------------------------------------------------------------------------------------------------------------------------|
+| `kind`               | string    | Y        | Type of payment method (i.e. `DEBIT_CARD`, `BITCOIN_ADDRESS`, `SQUARE_PAY`)                                                             |
+| `paymentDetails`     | object    | N        | An object containing the properties defined in an Offering's `requiredPaymentDetails` json schema. Excluded from the signature payload. |
+| `paymentDetailsHash` | string    | Y        | A hash [digest](#digest)  of `paymentDetails`                                                                                           |
```

Comment:
Same here. Not sure what a hash of a non-present `paymentDetails` would entail.

Reply:
Good catch -- typo on my part.